src/losses/Losses.jl

```diff
@@ -246,64 +246,66 @@ end
 function loss(
     lossType::LossH,
     H_pred::Matrix{F},
-    H_ref::Matrix{F},
+    H_ref,
+    V_ref, Vx_ref, Vy_ref,
     t::F,
     glacier,
     θ,
     simulation,
     normalization::F,
+    Δt,
 ) where {F <: AbstractFloat}
-    lH = loss(lossType.loss, H_pred, H_ref, normalization)
-    return lH
+    if isnothing(H_ref)
+        return 0.0
+    else
```

Comment:
Isn't a bit dangerous to have this here? If we call for the evaluation of a loss with `LossH` we should always provide with `H_ref`. I guess you are trying to be able to make this function call when the t where you stop does not have ice thickness data (but maybe it does have velocity data), but don't you think this can be included in other part of the code? If not, I will suggest having a nice comment here explaining why this `if isnothing(H_ref)`.

Reply:
Sounds good!